src/test/groovy/edu/ie3/datamodel/io/source/csv/CsvSystemParticipantSourceTest.groovy

```diff
@@ -159,7 +157,6 @@ class CsvSystemParticipantSourceTest extends Specification implements CsvTestDat
 
     where:
     operators               | types               | thermalBuses              || resultingSize | resultingSet
-    []                      | [sptd.hpInput.type] | [sptd.hpInput.thermalBus] || 1             | [new HpInput(sptd.hpInput.uuid, sptd.hpInput.id, OperatorInput.NO_OPERATOR_ASSIGNED, sptd.hpInput.operationTime, sptd.hpInput.node, sptd.hpInput.thermalBus, sptd.hpInput.qCharacteristics, sptd.emInput, sptd.hpInput.type)]
```

Comment:
I don't agree with just removing the successful cases completely. What is the problem here?

Reply:
In theses test we are explicitly testing cases that are leading to failures.

Reply:
Ok, I see. `resultingSize` and `resultingSet` is not used anymore and can be removed then